fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanTermVariationType.java

```diff
@@ -30,7 +30,8 @@ public enum LoanTermVariationType {
     GRACE_ON_INTEREST(7, "loanTermType.graceOnInterest"), //
     GRACE_ON_PRINCIPAL(8, "loanTermType.graceOnPrincipal"), //
     EXTEND_REPAYMENT_PERIOD(9, "loanTermType.extendRepaymentPeriod"), //
-    INTEREST_RATE_FROM_INSTALLMENT(10, "loanTermType.interestRateFromInstallment"); //
+    INTEREST_RATE_FROM_INSTALLMENT(10, "loanTermType.interestRateFromInstallment"), //
+    INTEREST_PAUSE(11, "loanTermType.interestPause"); //
```

Comment:
Add the new type to the `fromInt` method!

Reply:
Fixed